assets/sass/components/idea-hub/_googlesitekit-idea-hub-dashboard-cta.scss

```diff
@@ -19,13 +19,27 @@
 .googlesitekit-plugin {
 
 	.googlesitekit-idea-hub__dashboard-cta {
+		align-items: center;
 		background-color: $c-white;
-		display: flex;
+		height: 100%;
+		padding: $grid-gap-desktop;
+
+		@media (min-width: $bp-tablet) {
+			display: flex;
+		}
+
+		@media (min-width: $bp-desktop) {
+			padding: 48px;
+		}
 	}
 
 	.googlesitekit-idea-hub__dashboard-cta__content {
 		display: inline-block;
-		padding: $grid-gap-desktop;
+
+		@media (min-width: $bp-tablet) {
+			padding: $grid-gap-desktop 0 $grid-gap-desktop $grid-gap-desktop;
```

Comment:
@johnPhillips We can remove the padding top, bottom and right here since we just need the padding-left, to space the container from the icon.